cores/esp8266/Print.cpp

```diff
@@ -312,3 +304,8 @@ size_t Print::printFloat(double number, uint8_t digits) {
 
     return n;
 }
+
+int Print::availableForWrite() {
+    IAMSLOW();
+    return 1;
```

Comment:
according to comment in Arduino Print.h the default availableForWrite should return 0

> // default to zero, meaning "a single write may block"
    // should be overriden by subclasses with buffering
    virtual int availableForWrite() { return 0; }

Reply:
done